sdk/keyvault/keyvault-certificates/CHANGELOG.md

```diff
@@ -1,6 +1,18 @@
 # Release History
 
-## 4.2.0-beta.4 (Unreleased)
+## 4.2.0 (2021-06-08)
+
+### New Features
+
+- Added support for the 7.2 version of the Key Vault service API.
+
+### Bug Fixes
+
+- Fixed a bug with `beginDeleteCertificate` and `beginRecoverDeletedCertificate` in which unknown service errors wouldn't bubble up properly to the end users.
+- Fixed an issue where importing a certificate incorrectly required a Subject or Subject Alternative Name.
+- Fixed an issue where retrying a failed initial Key Vault request may result in an empty body.
```

Comment:
I believe we typically add in the PR# that fixed the bug as well so people can find background, etc..

(for example):
https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/servicebus/service-bus/CHANGELOG.md#bug-fixes-1

Reply:
I haven't seen it in Key Vault , so just been following the pattern here. It might be tricky to chase these now. I can start doing that going forward though, and go from there. Thanks for bringing it to my attention!